gateway/1.5/modules/ROOT/pages/policies-pdk-template-caching-policy.adoc

```diff
@@ -0,0 +1,424 @@
+= Data Caching Policy Template
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Use the data caching policy template as an example of how to xref:policies-pdk-configure-features.adoc#share-data-between-workers-and-caching.adoc[configure policy caching].
+
+== Before You Begin
+
+. xref:policies-pdk-prerequisites.adoc[]
+
+== Policy Use Case
+
+An antique store that buys and sells items needs to track the stock of each item in its collection and who buys it. To track the inventory movement, the company uses an API where each unique path represents a different item.. 
+
+During business hours, the catalog changes constantly and caching the available items isn't very useful. Outside of business hours, the stock is frozen, and the catalog can be cached without any concern. Given this, the manager of the antique store's server decides to implement a caching policy that caches the stock during the non-business hours. 
+
+The policy first checks the time of each incoming request. Then, if the request is outside of business hours and if a response for the path has not been previously cached, the policy caches the response. Then the policy returns the cached response for all following requests to the path outside of business hours. When business hours resume, the cache is cleared and is ready to cache new responses during the next period of non-business hours.
```

Comment:
How about putting this is a more scannable format? Since it's essentially steps, you could put it something like the following:

The server manager implements the caching policy as follows:

. The policy checks the time of each incoming request. 
. If the request is outside of business hours and if a response for the path has not been previously cached, the policy caches the response. 
. The policy returns the cached response for all following requests to the path outside of business hours. 
. When business hours resume, Anypoint Platform clears the cache and is ready to cache new responses during the next period of non-business hours.